include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -119,6 +120,19 @@ class SFML_AUDIO_API InputSoundFile
     ////////////////////////////////////////////////////////////
     unsigned int getSampleRate() const;
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the map of position in sample frame to sound channel
+    ///
+    /// This is used to map a sample in the sample stream to a
+    /// position during spatialisation.
+    ///
+    /// \return Map of position in sample frame to sound channel
+    ///
+    /// \see getSampleRate, getChannelCount, getDuration
+    ///
+    ////////////////////////////////////////////////////////////
+    std::vector<SoundChannel> getChannelMap() const;
```

Comment:
Can we make this return a `const std::vector<SoundChannel>&`? I see that `SoundRecorder::getChannelMap()` creates one on the spot, but maybe we could just cache it as a data member in `SoundRecorder` to support this more efficient API?

Reply:
Done.